_posts/2023-07-25-introducing-extensions-for-opensearch.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "Introducing extensions for OpenSearch"
+authors:
+  - vemsarat
+  - widdisd
+  - kazabdu
+  - dblock
+  - dagney
+  - minalsha
+  - kolchfa
+date: 2023-07-25
+categories:
+  - technical-posts
+Meta_keywords: OpenSearch extensions, OpenSearch plugins, OpenSearch extensibility
```

Comment:
These front-matter keywords are case sensitive. Can you change this to `meta_keywords` ?

Reply:
@nateynateynate Done. Thanks for catching it.